src/lib/io/readers.h

```diff
@@ -36,6 +36,8 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
 #define _READERS_h_
 
 namespace Partio{
+ParticlesDataMutable* readABC(  const char* filename,const bool headersOnly,std::ostream* errorStream);
+ParticlesDataMutable* readVDB(  const char* filename,const bool headersOnly,std::ostream* errorStream);
 ParticlesDataMutable* readBGEO(	const char* filename,const bool headersOnly,std::ostream* errorStream);
```

Comment:
note: this part would also nee some special ifdef handling if we made openvb + alembic optional.